ui/app/src/services/content.ts

```diff
@@ -1207,6 +1207,13 @@ export function unlock(site: string, path: string): Observable<boolean> {
   );
 }
 
+export function fetchWorkflowAffectedFiles(site: string, path: string): Observable<LegacyItem[]> {
```

Comment:
1. fetchWorkflowAffectedFiles => fetchWorkflowAffectedItems
2. parse to sandbox item using parseLegacyToSandbox...